2018/AsyncFromSyncIteratorContinuation.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var GetIntrinsic = require('../GetIntrinsic.js');
+var callBound = require('../helpers/callBound.js');
+
+var Call = require('./Call.js');
+var CreateIterResultObject = require('./CreateIterResultObject.js');
+var IteratorComplete = require('./IteratorComplete.js');
+var IteratorValue = require('./IteratorValue.js');
+
+var PromiseResolve = require('./PromiseResolve.js');
```

Comment:
```suggestion
var GetIntrinsic = require('../GetIntrinsic');
var callBound = require('../helpers/callBound');

var Call = require('./Call');
var CreateIterResultObject = require('./CreateIterResultObject');
var IteratorComplete = require('./IteratorComplete');
var IteratorValue = require('./IteratorValue');

var PromiseResolve = require('./PromiseResolve');
```
extensions should always be omitted.